bedrock/firefox/forms.py

```diff
@@ -10,8 +10,6 @@
 
 from lib.l10n_utils.dotlang import _lazy as _
 
-LANG_FILES = ['firefox/whatsnew-fx37']
-
```

Comment:
> Is this message actively used in any page?

I don't believe so, no. The send-to-device widget used different messaging for this error, which is in the macro template itself.